metaserver/src/main/scala/scala/meta/languageserver/search/SymbolIndex.scala

```diff
@@ -111,6 +112,21 @@ class SymbolIndex(
     }
   }
 
+  /** Returns the definition position of the symbol at the given position */
+  def references(
+      path: AbsolutePath,
+      line: Int,
+      column: Int
+  ): ReferencesResult = {
+    val locations = for {
+      symbol <- findSymbol(path, line, column).toSeq
```

Comment:
Note that `findSymbol` guards against symbols that have no definition, we should probably move that filter into the `definition` implementation.